apps/website/screens/components/data-grid/code/DataGridCodePage.tsx

```diff
@@ -167,7 +171,7 @@ const sections = [
           <tr>
             <td>uniqueRowId</td>
             <td>
-              <TableCode>string</TableCode>
+              <TableCode>string | number</TableCode>
```

Comment:
The type should be string only, but the type which the key has can be string or number.